packages/logger/src/formatter/LogFormatter.ts

```diff
@@ -112,6 +112,19 @@ abstract class LogFormatter implements LogFormatterInterface {
   #getDateFormatter = (timeZone: string): Intl.DateTimeFormat => {
     const twoDigitFormatOption = '2-digit';
 
+    if (!Intl.supportedValuesOf('timeZone').includes(timeZone)) {
```

Comment:
Nice one!

I would extract the value in a variable to avoid extra boilerplate and do this:

```ts
const validTimeZone = Intl.supportedValuesOf('timeZone').includes(timeZone) ? timeZone : 'UTC';
```

and then modify the existing `new Intl.DateTimeFormat()` call below to use `validTimeZone`.